### PR TITLE
Tekton Results Working Group

### PR DESCRIPTION
**Pull request by adambkaplan**
Scaffold the working group call for Tekton Results.

### CODE REVIEW

File: working-groups.md [hunk 1]
```diff
@@ -399,6 +400,38 @@ TBD [community#629](https://github.com/tektoncd/community/issues/629)).
...
 
+## Results
+
+This is the working group for [tektoncd/results](https://github.com/tektoncd/results).
+This should not be confused with the [TEP-0086 Larger Results](#tep-0086-larger-results) working group.
+
+| Artifact | Link                                                              |
+|----------|-------------------------------------------------------------------|
+| Forum    | [tekton-dev@](https://groups.google.com/forum/#!forum/tekton-dev) |
+| Meeting Notes              | TBD |
```
Comment (adambkaplan) [inline working-groups.md:411]: We'll need to set up a shared Google Doc for the combined meeting notes.

File: working-groups.md [hunk 2]
```diff
@@ -399,6 +400,38 @@ TBD [community#629](https://github.com/tektoncd/community/issues/629)).
...
+| Slack Channel              | [#results](https://tektoncd.slack.com/messages/results) |
+
+### Europe + North America
+
+| Artifact | Link                                                              |
+|----------|-------------------------------------------------------------------|
+| Community Meeting VC       | TBD |
```
Comment (adambkaplan) [inline working-groups.md:418]: Needed: link to Zoom meeting

File: working-groups.md [hunk 3]
```diff
@@ -399,6 +400,38 @@ TBD [community#629](https://github.com/tektoncd/community/issues/629)).
...
+| Community Meeting Calendar | Thursday every 4 weeks, 9:30a PST (alternates with Europe + Asia)<br>Link TBD|
+
+| &nbsp;                                                     | Facilitators    | Company | Profile                                       |
+|------------------------------------------------------------|-----------------|---------|-----------------------------------------------|
+| <img width="30px" src="https://github.com/adambkaplan.png">| Adam Kaplan     | Red Hat | [adambkaplan](https://github.com/adambkaplan) |
```
Comment (adambkaplan) [inline working-groups.md:423]: @dibyom I imagine you'll want to be on the facilitator rotation for the Europe + NA meeting?

File: working-groups.md [hunk 4]
```diff
@@ -399,6 +400,38 @@ TBD [community#629](https://github.com/tektoncd/community/issues/629)).
...
+
+### Europe + Asia
+
+| Artifact | Link                                                              |
+|----------|-------------------------------------------------------------------|
+| Community Meeting VC       | TBD |
+| Community Meeting Calendar | Thursday every 4 weeks, 5:30a PST (alternates with Europe + NA)<br>Link TBD|
+
+| &nbsp;                                                     | Facilitators    | Company | Profile                                     |
+|------------------------------------------------------------|-----------------|---------|---------------------------------------------|
```
Comment (adambkaplan) [inline working-groups.md:433]: Need facilitators who are willing to run the meetings for EMEA + Asia.

File: working-groups.md (hunk 3 above)
Comment (dibyom) [reply working-groups.md:423]: Sure, I can help out.

Comment (dibyom) [reply working-groups.md:423]: Maybe @alan-ghelardi will be interested as well?

**Comment by adambkaplan** [comment]
A thought here that we should have the meetings alternate weekly instead of every other week. This can help with "pull request wrangling"

File: working-groups.md (hunk 4 above)
Comment (avinal) [reply working-groups.md]: Avinal Kumar | Red Hat

**Comment by adambkaplan** [comment]
@wlynch @bobcatfish @dibyom removed the WIP, we are ready to go with the Results WG!

**Review by dibyom — APPROVED** [review]

**Comment by vdemeester** [comment]
@adambkaplan needs a rebase 🙏🏼

**Comment by afrittoli** [comment]
@adambkaplan this is still waiting for some love (rebase) 🙏

**Comment by adambkaplan** [comment]
@afrittoli @vdemeester _finally_ got around to rebasing this PR. Took me long enough.

fyi @dibyom @bobcatfish I'm proposing we drop the NA/EU working group meeting. This has been sparsely attended, and the past 3 meetings were cancelled due to lack of attendance.

**Pull request merged**